Misc/NEWS.d/next/C API/2021-09-19-17-18-25.bpo-44687.3fqDRC.rst

```diff
@@ -0,0 +1 @@
+Add a new :c:macro:`PEEK_CHECK_CLOSED` macro to check if a file is closed using a method specific to BufferedReader.peek()
```

Comment:
The NEWS entry should be relevant to users, not developers. This would be a better commit message, while the PR title would be better for NEWS.

Reply:
thanks for the feedback, i've updated the NEWS entry to incorporate more of the PR title. Hope that works :+1: